stix_shifter_modules/tanium/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,1013 @@
+{
+    "id": 
+    [
+        {
+            "key": "x-ibm-finding.finding_type",
+            "object": "alert",
+            "transformer": "ReturnAlertTransformer"
+        },
+        {
+            "key": "x-ibm-finding.alert_id",
+            "object": "alert"
+        }
+    ],
+    "eid": 
+    {
+        "key": "x-tanium-alert.eid",
+        "object": "tanium-alert"
+    },
+    "type":
```

Comment:
can we map this to `x-ibm-finding.finding_type` ? i see you have mapped that for "id" field. but if there's already a type field maybe map this to "finding_type"